tests/core/core.spec.ts

```diff
@@ -484,27 +484,40 @@ describe("@dataform/core", () => {
   });
 
   describe("compilers", () => {
-    it("extract_blocks", function() {
+    it("extract_blocks", function () {
       const TEST_SQL_FILE = `
         /*js
         var a = 1;
         */
         /*js
         var c = 3;
         */
-        --js var b = 2;
         /*
         normal_multiline_comment
         */
+        -- --js var x = 1200;
+        --js var b = 2;
```

Comment:
nit: did this have to move?

Reply:
yes, just to make sure it works when some `js` code is inside.